### DISCUSSION TITLE
No matching flow disconnect client

### DISCUSSION THREAD

**Discussion by Nathanwoodburn** [engity-com/bifroest#66] [Ideas]
G'day,

When I try to login with something that doesn't match any flow (all other flows require a non matched regex), I get dropped into a password prompt. It would be nice to have the option to just disconnect the connection instead

**Comment by blaubaer** [comment]
Hi @Nathanwoodburn,

that is a good proposal. I also experienced this issue. So far I found no solution for that because the client is more or less controlling this behavior by simply retrying the authorization mechanisms, but I also have to admin that I didn't tried hard enough to maybe find a good (not thaaaat hacky) solution for this.

I've created a ticket for this: https://github.com/engity-com/bifroest/issues/67

**Reply by Nathanwoodburn** [reply]
Yeah no rush or anything. Just will make it a bit easier to debug stuff

**Reply by blaubaer** [reply]
I see. Generally: Does the log output help, especially if you switch to [`--log.level=DEBUG`](https://bifroest.engity.org/reference/cli/#flag-log.level)?

**Reply by Nathanwoodburn** [reply]
:) didn't even notice the log options until after I fixed my problem

**Reply by blaubaer** [reply]
I see. Next time you know it. ;-)
